lib/cgrates_web_jsonapi/ets_cache.ex

```diff
@@ -0,0 +1,37 @@
+defmodule CgratesWebJsonapi.EtsCache do
+  @moduledoc """
+  Cache via ETS.
+  """
+
+  def get(args, opts \\ []) do
```

Comment:
1. Add typespec plz
2. args should be a function because you should execute SQL request only if there is no value in the cache